app/assets/stylesheets/opengraph.scss

```diff
@@ -4,10 +4,10 @@
   text-decoration: none;
   font-style: normal;
   margin: 10px 0px 10px 0px;
-  border-top: solid 1px $border-grey;
-  border-bottom: solid 1px $border-grey;
-  padding: 10px 0px 5px 0px;
+  border: solid 1px #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.